ng-dev/caretaker/check/github.ts

```diff
@@ -93,7 +93,7 @@ export class GithubQueriesModule extends BaseModule<GithubQueryResults | void> {
         {
           type: 'ISSUE',
           first: MAX_RETURNED_ISSUES,
-          query: `"${repoFilter} ${query.replace(/"/g, '\\"')}"`,
+          query: `"${repoFilter} ${query.replace(/\"/g, '\\"')}"`,
```

Comment:
I think this is incorrect. Maybe something like?

```suggestion
          query: `"${repoFilter} ${query.replace(/([\\"])/g, '\\$1')}"`,
```

Reply:
I think what you have here will replace `\"` instead of replacing `"` with `\"`

Maybe I am misreading it though?

Reply:
I thought the security concern was that we do not escape a `\` directly. Not `\"`

Reply:
Since we are trying to replace all `"` with `\"` I think its the failure to escape the `"` character

Reply:
The error is `This does not escape backslash characters in the input.`. We can just try what you have, but essentially `\"` is a noop and just turns into `"` I think.